gammapy/data/event_list.py

```diff
@@ -393,6 +393,58 @@ def peek(self):
         plt.tight_layout()
         plt.show()
 
+    def peek_time_map(self, num_sides=0):
```

Comment:
Can you take a matplotlib axes object as input and also return it, i.e. use the common pattern for plotting functions:
[example](https://gammapy.readthedocs.org/en/latest/api/gammapy.time.plot_fermi_3fgl_light_curve.html), [other example](https://gammapy.readthedocs.org/en/latest/_modules/gammapy/irf/effective_area_table.html#EffectiveAreaTable2D.plot_image)?

This will make it easier to put this plot on one panel in a larger Figure.